app/src/main/java/com/example/unlibrary/library/LibraryViewModel.java

```diff
@@ -1,24 +1,49 @@
 package com.example.unlibrary.library;
 
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.Transformations;
 import androidx.lifecycle.ViewModel;
 
 import com.example.unlibrary.models.Book;
 
 import java.util.ArrayList;
 
+/*
+ * LibraryViewModel
+ *
+ * October 22, 2020
+ *
+ * Copyright (c) Team 24, Fall2020, CMPUT301, University of Alberta
+ */
+
+/**
+ * Manages the Library flow business logic. Connects the library fragments to the repository.
+ */
 public class LibraryViewModel extends ViewModel {
-    private ArrayList<Book> mBooks;
+    private LiveData<ArrayList<Book>> mBooks;
+    private LibraryRepository libraryRepository;
```

Comment:
```suggestion
    private LibraryRepository mLibraryRepository;
```
same here for private variables